src/main.go

```diff
@@ -99,14 +116,15 @@ func main() {
     replicas: *replicas,
     subvolumes: *subvolumes,
     protect: *protect,
+    htpasswdfile: htpasswdfile,
   }
 
   if command == "server" {
     http.ListenAndServe(fmt.Sprintf(":%d", *port), &a)
   } else if command == "rebuild" {
-    a.Rebuild()
+    a.Rebuild(userpassValue)
```

Comment:
Why is this one special, there's tons of other options, and they aren't passed in like this.